SentinelHub/tests/test_qgis_environment.py

```diff
@@ -0,0 +1,37 @@
+# coding=utf-8
+import os
+
+import pytest
+
+pytest.importorskip("qgis.core")
+
+from qgis.core import QgsCoordinateReferenceSystem, QgsProviderRegistry, QgsRasterLayer  # noqa: E402
+
+CRS = "EPSG:4326"
+WKT = (
+    'GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",'
+    'SPHEROID["WGS_1984",6378137.0,298.257223563]],'
+    'PRIMEM["Greenwich",0.0],UNIT["Degree",'
+    "0.0174532925199433]]"
+)
+
+
+def test_providers_registry() -> None:
+    r = QgsProviderRegistry.instance()
+    assert "gdal" in r.providerList()
+    assert "ogr" in r.providerList()
+
+
+def test_projection_reading() -> None:
+    crs = QgsCoordinateReferenceSystem()
+    crs.createFromWkt(WKT)
+    auth_id = crs.authid()
+    assert auth_id == CRS
```

Comment:
Any reason for not doing it like this? (same suggestion in `test_loaded_raster`)

```suggestion
    assert crs.authid() == CRS
```

Reply:
No, you are right, it doesn't add information. Its cleaner in one line. I' ll fix it.